R6SDataReader.py

```diff
@@ -82,8 +118,17 @@ def main(filename: str, userfile: str, video: str) -> int:
         actualKiller = find_closest_username(killer, usernames)
         actualVictim = find_closest_username(victim, usernames)
         if actualKiller != None and actualVictim != None:
-            killCount += 1
-    print(killCount)
+            # adding the kill to the list
+            addKill(killboxes.getRound(), actualKiller, actualVictim, killboxes.getTimestamp, kill_list)
```

Comment:
killboxes.getRound and getTimestamp gets the image croppings for the picture itself. I was having trouble getting those parts to be read correctly since the text is surrounded by the game itself, making the background inconsistent. We could save those images directly and insert them into an excel spreadsheet, in which case this would be correct (minus the killboxes.getTimestamp -> killboxes.getTimestamp())
```suggestion
            addKill(killboxes.getRound(), actualKiller, actualVictim, killboxes.getTimestamp(), kill_list)
```